### PR TITLE
chore: upgrade graalvm docker image to use 22.2.0

### PR DESCRIPTION
**Pull request by mpeddada1**

### CODE REVIEW

**Comment by mpeddada1** [comment]
We will need to merge this in when the [shared-dependencies 3.0.5](https://github.com/googleapis/java-shared-dependencies/pull/880) upgrade PRs show up in downstream libraries.

**Review by suztomo — APPROVED** [review]

File: java/graalvm/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.1.0
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.2.0-b1
```
Comment (suztomo) [inline java/graalvm/Dockerfile:15]: Memo: What's "-b1"? It's written in https://github.com/graalvm/container/pkgs/container/graalvm-ce:

> The version tag defines the level of specificity. It is recommended that the most specific tag be used, e.g., java17-21.3.0 or java17-21.3.0-b1, where the -b1 means the image required a patch and this specific build will never change

**Comment by suztomo** [comment]
If a repository hasn’t received Release Please PR, you can create one.

**Comment by suztomo** [comment]
Should we use a tag when specifying the container image rather than “latest”? With such set up, we can create one pull request that upgrades the version of both shared dependencies BOM and the container image together.

**Comment by mpeddada1** [comment]
Nice, having the shared-dependencies upgrade and the container image upgrade in one PR would be ideal! Are we referring to tagging the testing-docker image in the native image Kokoro job config?

**Comment by suztomo** [comment]
Yes, tagging the container at the time of publishing.

**Comment by mpeddada1** [comment]
That is a good idea! Will be useful for the upgrade to 22.3. Created an issue to track this: #253.

**Review by meltsufin — APPROVED** [review]

**Pull request merged**